ui/perfherder/alerts/AlertHeader.jsx

```diff
@@ -93,17 +108,28 @@ const AlertHeader = ({ alertSummary, repoModel, issueTrackers }) => {
             ) : (
               { bugNumber }
             )}
-          </span>
+            <span>·</span>
+          </Col>
         )}
-      </span>
-    </div>
+        <Col className="p-0" xs="auto">
+          <Assignee
+            assigneeUsername={alertSummary.assignee_username}
+            onAssigneeChange={updateAssignee}
```

Comment:
I find it less confusing and easier to trace props if they are named the same as they passed from component to component.

Reply:
Yes, that's true.